class/defaults.yml

```
@@ -30,6 +30,18 @@ parameters:
       projectsyn: syn-*
       appuio: appuio-*
 
+    allowedNamespaceLabels:
+      kubernetesGenerated:
+        - kubernetes.io/metadata.name
+      appuioUser: user.appuio.io/*
+
+    allowedNamespaceAnnotations:
+      kubernetesGenerated:
+        - kubectl.kubernetes.io/last-applied-configuration
+      appuioUser: user.appuio.io/*
```

Actually, this might be confusing/conflicting with the term `User` from the Control API (https://kb.vshn.ch/appuio-cloud/references/architecture/control-api.html)
I propose to use a different domain
 ```suggestion
      appuioUserDefined: custom.appuio.io/*
```

---

@corvus-ch @tobru wdyt?

---

I'd go with the suggestion from @ccremer